app/src/components/transactions/TransactionIcon.vue

```diff
@@ -13,9 +14,14 @@ export default {
       required: true,
     },
   },
+  data: () => ({
+    isExtension: config.isExtension,
+  }),
   computed: {
     icon() {
-      return `/img/icons/activity/${this.transactionType}.svg`
+      return this.isExtension
```

Comment:
This should be handled by webpack. The paths should be the same.

Reply:
yes, I know, the images don't display :thinking:  but I was seeing this ugly `ERR FILE NOT FOUND` and I thought, let's fix this

Reply:
ok, so then I will simply create an `img/icons` folder in extension and that should fix it too. No need to use webpack for that then

Reply:
I mean the images shouldn't be in our repo twice. Webpack should copy them in the right place

Reply:
O.o do yo need to rebuild the whole thing when you change the `webpack.config`? I think you do :sweat:

Reply:
yep